### PR TITLE
Add webhook information for OpenShift

### PR DESCRIPTION
**Pull request by brian-avery**
OpenShift requires enabling webhooks in the master-config.yaml in order to use automatic injection.

### CODE REVIEW

File: content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md [hunk 1]
```diff
@@ -39,3 +39,31 @@ constraints as part of sidecar injection:
 {{< text bash >}}
 $ oc adm policy add-scc-to-user privileged -z default -n <target-namespace>
 {{< /text >}}
+    
+## Automatic Injection
+Webhook and certificate signing requests support must be enabled for auto injection to work. To do so, you must modify the master configuration file for the cluster.
```
Comment (sdake) [inline content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md]: auto->automatic.  Please link to the automatic injection page.  Remove "To do so".

File: content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md [hunk 2]
```diff
@@ -39,3 +39,31 @@ constraints as part of sidecar injection:
 {{< text bash >}}
 $ oc adm policy add-scc-to-user privileged -z default -n <target-namespace>
 {{< /text >}}
+    
+## Automatic Injection
+Webhook and certificate signing requests support must be enabled for auto injection to work. To do so, you must modify the master configuration file for the cluster. 
+
+On the master node in the directory containing the master configuration file (likely /etc/origin/master/master-config.yaml), create a file named master-config.patch with the following contents:
```
Comment (sdake) [inline content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md]: use an tip callout to indicate the default path.  See https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts for more details.

File: content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md [hunk 3]
```diff
@@ -39,3 +39,31 @@ constraints as part of sidecar injection:
...
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+    ValidatingAdmissionWebhook:
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+{{< /text >}}
+In the same directory, execute:
```
Comment (sdake) [inline content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md]: need CR between 61 and 62.

**Comment by sdake** [comment]
Brian,

Please fix linting errors in content.  To link locally - you can run `make lint` in the repository.

Here are the linting errors: https://circleci.com/gh/istio/istio.io/10110?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md (hunk 2 above)
Comment (brian-avery) [reply content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md]: Thanks. Updated.

File: content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md (hunk 1 above)
Comment (brian-avery) [reply content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md]: Thanks. Updated.

**Pull request merged**